components/Content/Article.jsx

```diff
@@ -39,24 +39,29 @@ const Article = ({ article, href, showReadTime, showDate }) => {
 
   return (
     <Link href={href}>
-      <article className={CARD_CLASS}>
+      <article className={`${CARD_CLASS} h-full overflow-hidden`}>
         {!imageError && (url || width || height) ? (
-          <Image
-            src={imageDomain + url}
-            width={width}
-            height={height}
-            alt={article.attributes.title}
-            className="rounded-t-lg object-cover"
-            onError={() => {
-              setImageError(true);
-            }}
-          />
+          <div className="flex h-full">
+            <Image
+              src={imageDomain + url}
+              width={width}
+              height={height}
+              alt={article.attributes.title}
+              className="rounded-t-lg py-auto object-contain"
+              onError={() => {
+                setImageError(true);
+              }}
+            />
+          </div>
         ) : (
-          <div style={{ height: 200 }} className="bg-gray-100 text-gray-500" />
+          <div
+            style={{ height: 200 }}
```

Comment:
can we just use `h-[200px]` in class name? and maybe we don't need `xl:min-h-[200px]` there